src/mapping/dontbuymeme.ts

```diff
@@ -0,0 +1,36 @@
+import { Transfer } from "../../generated/templates/NftContract/ERC721";
+import { Nft, NftContract } from "../../generated/schema";
+import { fetchName, fetchSymbol, handleTransfer } from "./mapping";
+import { DontBuyMeme } from "../../generated/DontBuyMeme/DontBuyMeme";
+import { DONTBUYMEME, ZERO_ADDRESS } from "../constants";
+import { Address, BigInt, log } from "@graphprotocol/graph-ts";
+
+export function handleTransferDontBuyMeme(event: Transfer): void {
+  let address = event.address.toHexString();
+  if (NftContract.load(address) == null) {
+    let nftContract = new NftContract(address);
+    nftContract.name = fetchName(event.address);
+    nftContract.symbol = fetchSymbol(event.address);
+    nftContract.platform = "DontBuyMeme";
+    nftContract.save();
+  }
+
+  handleTransfer(event);
```

Comment:
Not sure if this code gonna work, as there is no `tokenURI` method in dontbuymeme contract. They have `uri` instead`. Could you please refactor URI parsing, so it will work for all platforms?